source/Octopus.Tentacle/Configuration/TentacleConfiguration.cs

```diff
@@ -97,10 +97,8 @@ public string PackagesDirectory
             get { return Path.Combine(ApplicationDirectory, "Packages"); }
         }
 
-        public string JournalFilePath
-        {
-            get { return Path.Combine(home.HomeDirectory ?? ".", "DeploymentJournal.xml"); }
-        }
+        public string JournalFilePath => Path.Combine(home.HomeDirectory ?? ".", "DeploymentJournal.xml");
+        public string PackageRetentionJournalPath => Path.Combine(home.HomeDirectory ?? ".", "PackageRetentionJournal.json");
```

Comment:
I went with `PackageRetentionJournalPath` instead of `PackageRetentionJournalFilePath` to match with the environment variable name even though the existing tentacle journal is called `JournalFilePath`. I'm open to using either name but `File` in the name doesn't seem necessary to me.